### PR TITLE
RGW: realm get default command should show the realm name along with id

### PR DESCRIPTION
**Pull request by shreyanshjain7174**
`radosgw-admin realm get default` command should show the realm name along with id of the default realm. 

Fixes: https://tracker.ceph.com/issues/64888

Signed-off-by: Shreyansh Sancheti <ssanchet@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -4855,13 +4856,15 @@ int main(int argc, const char **argv)
 	  return -ret;
 	}
 	cout << "default realm: " << default_id << std::endl;
+  cout << "default realm name: " << default_name << std::endl;
```
Comment (thotz) [inline src/rgw/rgw_admin.cc:4859]: Both log messages can be combined

File: src/rgw/rgw_admin.cc [hunk 2]
```diff
@@ -4846,7 +4846,8 @@ int main(int argc, const char **argv)
     case OPT::REALM_GET_DEFAULT:
       {
 	string default_id;
-	int ret = cfgstore->read_default_realm_id(dpp(), null_yield, default_id);
+  string default_name;
```
Comment (thotz) [inline src/rgw/rgw_admin.cc:4849]: indentation issue??

**Review by thotz — CHANGES_REQUESTED** [review]
Some minor nits, have check realm rename scenarios for this PR??

**Comment by shreyanshjain7174** [comment]
jenkins test api
jenkins test make check

File: src/rgw/rgw_zone_types.h [hunk 3]
```diff
@@ -85,16 +85,19 @@ WRITE_CLASS_ENCODER(RGWNameToId)
 
 struct RGWDefaultSystemMetaObjInfo {
   std::string default_id;
+  std::string default_name;
 
   void encode(bufferlist& bl) const {
     ENCODE_START(1, 1, bl);
```
Comment (mattbenjamin) [inline src/rgw/rgw_zone_types.h:91]: doesn't this need a version increment?

**Pull request closed without merging**